### PR TITLE
Add tests for variables

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test]

File: plans/variables_plan.fmf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+summary: Basic variables test
+execute:
+    how: tmt
+    script: |
+        final_result=0
+        # Test if variaable $SOME_DUMMY_VARIABLE is 'foo_bar_variable'
```
Comment (zmiklank) [inline plans/variables_plan.fmf:6]:
```suggestion
        # Test if variable $SOME_DUMMY_VARIABLE is 'FOO_BAR_VARIABLE'
```

File: plans/variables_plan.fmf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        if [ "$SOME_DUMMY_VARIABLE" != "FOO_BAR_VARIABLE" ]; then
+            final_result=1
+        fi
+        # Test if variable $OS_VARIABLE is 'DUMMY_OS'
+        if [ "$OS_VARIABLE" == "DUMMY_OS" ]; then
```
Comment (zmiklank) [inline plans/variables_plan.fmf:11]:
```suggestion
        if [ "$OS_VARIABLE" != "DUMMY_OS" ]; then
```

File: .github/workflows/variables_test.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          ref: "refs/pull/${{ github.event.issue.number }}/head"
+
+      - name: Run the tests
+        uses: ./
+        with:
+          api_key: ${{ secrets.TF_PUBLIC_API_KEY }}
+          git_url: "https://github.com/sclorg/testing-farm-as-github-action"
+          git_ref: "main"
+          pull_request_status_name: "Variables test"
+          variables: "SOME_DUMMY_VARIABLE:FOO_BAR_VARIABLE;OS_VARIABLE=DUMMY_OS"
```
Comment (zmiklank) [inline .github/workflows/variables_test.yml:28]:
```suggestion
          variables: "SOME_DUMMY_VARIABLE=FOO_BAR_VARIABLE;OS_VARIABLE=DUMMY_OS"
```

File: .github/workflows/variables_test.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  container-tests:
+    runs-on: ubuntu-22.04
+    name: "Smoke test for testing farm as a github action"
```
Comment (zmiklank) [inline .github/workflows/variables_test.yml:8]:
```suggestion
    name: "Variables test for testing farm as a github action"
```

**Review by zmiklank — CHANGES_REQUESTED** [review]

**Pull request merged**